Tools/ssl/multissltests.py

```diff
@@ -160,30 +127,55 @@
 )
 
 
-class AbstractBuilder(object):
-    library = None
-    url_templates = None
-    src_template = None
-    build_template = None
+class AbstractBuilder(object, metaclass=ABCMeta):
```

Comment:
```suggestion
class AbstractBuilder(ABC):
```

no need for ABCMeta.